src/jvmMain/kotlin/processing/GenerateOutput.kt

```diff
@@ -16,6 +18,18 @@ fun generateOutput(
     println(outputDirPath)
     println(columnNamesPath)
 
+    //This should probably be changes and the expenseLogPath2 should have some sort of nullable value
+    //or change this to have it be a lateInit...
+    val fileParser = FileInputParser(expenseLogPath1, expenseLogPath2)
```

Comment:
If this needs to be changed the easiest way would be to check if expenseLogPath2 is empty and pass null if it is. 
Something like this should work:
```suggestion
    val fileParser = if (expenseLogPath2.trim().isEmpty())
        FileInputParser(expenseLogPath1, null) 
    else 
        FileInputParser(expenseLogPath1, expenseLogPath2) 
```
Not sure if you want to change it now or are just marking it with the comment for a later pr.